runtime/doc/editing.txt

```diff
@@ -1236,29 +1244,50 @@ present in 'cpoptions' and "!" is not used in the command.
 							*:chd* *:chdir*
 :chd[ir][!] [path]	Same as |:cd|.
 
+							*:tc* *:tcd*
+:tc[d][!] {path}        Like |:cd|, but set the current directory for the
+                        current tab.  The current directory for other tabs
+                        or the local directory of windows in this tab is not
+                        changed, except for the current window.
+
+							*:tcd-*
+:tcd[!] -		Change to the previous current directory (before the
+			previous ":tcd {path}" command).
+
+							*:tch* *:tchdir*
+:tch[dir][!]		Same as |:tcd|.
+
 							*:lc* *:lcd*
 :lc[d][!] {path}	Like |:cd|, but only set the current directory for the
 			current window.  The current directory for other
-			windows is not changed.
+			windows or any tabs is not changed.
```

Comment:
I had suggested in another comment to return a number: Vim already returns `0` for no local directory and `1` for window-local directory. My suggestions was to tread this like a bitfield: `2` for tab-local directory and `3` for window- and tab-local directory:
~~~~
0b00 " nothing
0b01 " window
0b10 " tab
0b11 " both
~~~~
It's backwards-compatible with the current implementation. The only problem I see is if plugin authors are testing the result of `hastabdir()` like a boolean instead of the number `1`. In that case the plugins would tread the number `2` as having a window-local directory even if it hasn't. And testing for `1` explicitly would fail for `3` even though there is a window-local directory. We are working with a running system so we must take this into consideration.